ddtrace/contrib/cassandra/__init__.py

```diff
@@ -28,7 +28,9 @@
 
 with require_modules(required_modules) as missing_modules:
```

Comment:
I think we accidentally deleted the `ddtrace/contrib/cassandra/patch.py` file: https://github.com/DataDog/dd-trace-py/blob/main/ddtrace/contrib/cassandra/patch.py. Can we add it back and follow the deprecation pattern we used in other integration this should fix the failing test